polkadot/node/test/service/src/chain_spec.rs

```diff
@@ -111,8 +111,12 @@ fn polkadot_testnet_genesis(
 	const ENDOWMENT: u128 = 1_000_000 * DOTS;
 	const STASH: u128 = 100 * DOTS;
 
-	// Prepare node features with V2 receipts enabled.
+	// Prepare node features with V2 receipts
+	// and elastic scaling enabled.
 	let mut node_features = NodeFeatures::new();
+	node_features.resize(node_features::FeatureIndex::ElasticScalingMVP as usize + 1, false);
```

Comment:
You just need to resize only once to `FeatureIndex::FirstUnassigned`. You are now doing it twice.

Reply:
I have made an update. Let me know if this aligns.